sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt

```diff
@@ -0,0 +1,48 @@
+package com.badoo.reaktive.samplemppmodule.binder
+
+import com.badoo.reaktive.disposable.CompositeDisposable
+import com.badoo.reaktive.observable.map
+import com.badoo.reaktive.observable.subscribe
+import com.badoo.reaktive.samplemppmodule.store.KittenStoreBuilder
+import com.badoo.reaktive.samplemppmodule.view.KittenView
+import com.badoo.reaktive.utils.atomic.AtomicReference
+
+class KittenBinder(
+    storeBuilder: KittenStoreBuilder
```

Comment:
Why we can't pass store directly?

Reply:
Because client will be responsible to create it. I wanted to make it internal. Store also has dependencies, they are internal as well. The idea is you create StoreBuilder and pass any client specific dependencies (currently none). And StoreBuilder then creates a Store using your dependencies as well as internal stuff (like Loader).